### DISCUSSION TITLE
指令触发器的用法

### DISCUSSION THREAD

**Discussion by SakuraAzusa** [FloatTech/ZeroBot-Plugin#148] [Q&A] [answered]
因为没有相关知识，看到指令触发器job有个想法，假如我想让他给特定的某个号签到，大致内容就应该是这样
```
记录在"30 8 * * *"触发的指令
签到 + uid=****
```
那么特定的号码应该怎么输入？

**Comment by kanrichan** [comment, marked as answer]
是指这个uid是动态的吗

**Reply by SakuraAzusa** [reply]
> 是指这个uid是动态的吗

uid指的是QQ号，因为我看`score`插件里的`model.go`请求的数据是uid，那么我想让bot在每天特定的时间给qq123456签到应该怎么输入？

**Reply by kanrichan** [reply]
就案例来说，暂时只能给设置 `job` 的人定时签到

**Reply by SakuraAzusa** [reply]
了解了

**Reply by fumiama** [reply]
因为job是代表自己执行的任务，所以如果可以随意指定触发者的话，就乱套了。
